tests/Block/AdminSearchBlockServiceTest.php

```diff
@@ -35,12 +36,19 @@ class AdminSearchBlockServiceTest extends BlockServiceTestCase
      */
     private $searchHandler;
 
+    /**
+     * @var TemplateRegistryInterface
+     */
+    private $templateRegistry;
+
     protected function setUp(): void
     {
         parent::setUp();
 
         $this->pool = $this->createMock(Pool::class);
         $this->searchHandler = $this->createMock(SearchHandler::class);
+        $this->templateRegistry = $this->prophesize(TemplateRegistryInterface::class);
```

Comment:
Since other mock are not using, I'll recommend to be consistent

```suggestion
        $this->templateRegistry = $this->createMock(TemplateRegistryInterface::class);
```